draft-ietf-rtcweb-jsep.xml

```diff
@@ -1562,6 +1563,70 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           results in connectivity being established.</t>
         </section>
       </section>
+      <section title="RtpTransceiver" anchor="sec.transceiver">
+        <section title="stop" anchor="sec.transceiver-stop">
+          <t>
+            The stop method stops an RtpTransceiver.  This
+            will cause future calls to createOffer to generate a zero
+            port for the associated m= section.  See below for more
+            details.
+          </t>
+        </section>
+        <section title="isStopped" anchor="sec.transceiver-stopped">
+          <t>
+            The isStopped method returns "true" if the transceiver has
+            been stopped, either by a call to stopTransceiver or by
+            applying an answer that rejects the associated m= section, and
+            "false" otherwise.
+          </t>
+          <t>
+            A stopped RtpTransceiver does not send any
+            outgoing RTP or RTCP or process any incoming RTP
+            or RTCP.  It cannot be restarted.
+          </t>
+        </section>
+        <section title="setDirection" anchor="sec.transceiver-set-direction">
+          <t>
+            The setTransceiverDirection method sets the
+            direction of a transceiver, which affects the
+            direction attribute of the associated m= section on
+            future calls to createOffer and createAnswer.
+          </t>
+        </section>
+        <section title="setCodecPreferences" anchor="sec.transceiver-set-codec-preferences">
+          <t>
+            The setCodecPreferences method sets the codec
+            preferences of a transceiver, which affect the
```

Comment:
Changed to "in turn affect" to make it unambiguous that this means "the preferences affect".